### PR TITLE
text.chained accept varargs

### PR DESCRIPTION
**Pull request by EugeneDar**
Closes: https://github.com/objectionary/eo-strings/issues/120

### CODE REVIEW

**Review by Graur — APPROVED** [review]
@EugeneDar Looks good, thanks!
@yegor256 Please, have a look

**Review by Graur — CHANGES_REQUESTED** [review]

File: src/main/eo/org/eolang/txt/text.eo [hunk 1]
```diff
@@ -397,11 +397,16 @@
         b
 
   # Returns concatenaion of two strings
```
[the remark this answers is not in the document]
Comment (EugeneDar) [reply src/main/eo/org/eolang/txt/text.eo:399]: @Graur my fault, I'll fix it.

**Comment by EugeneDar** [comment]
@Graur please, have a look again

**Review by Graur — APPROVED** [review]
@EugeneDar Looks good, thanks!
@yegor256 Please, take a look

**Pull request merged**